.vib/discourse/goss/discourse.yaml

```diff
@@ -0,0 +1,17 @@
+command:
```

Comment:
I would add several checks related to the logic executed at compilation time:

- Check the global npm package `terser` installed
- Check Passenger native support works is properly installed

Reply:
I added the `terser` node module test:
```
  check-terser-modules:
    exec: npm list -g
    exit-status: 0
    stdout:
      - terser
```

Is it okay or would it be enough if we simply add it to the list of checked binaries (as it should be in the PATH)? Maybe both?

Reply:
I think the node module test is great 👍